app/views/drops/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @drops.each do |drop| %>
+<%= drop.id %>
+<%= drop.title%>
+</br>
+<% end %>
```

Comment:
woot! new page!
It would be great to have links here so that ppl can be redirected to the appropriate drop page.